src/pbft-core/config.go

```diff
@@ -41,10 +42,21 @@ type Config struct {
 	NumKeys   int      // NumKeys is the count of IP addresses (BFT nodes) participating
 }
 
+// LoadConfig ini file
+func LoadConfig() (configData *ini.File, err error) {
+	configData, err = ini.Load("/home/abhishek/truechain/cfg.ini")
```

Comment:
I'm sure you're already working on eliminating this hardcoded path to the cfg.ini file.

Reply:
@samikshan yes.

Reply:
@samikshan I have updated the PR.